app/views/partials/GitRepositoryTeamNames.scala.html

```diff
@@ -0,0 +1,32 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.cataloguefrontend.connector.GitRepository
+@import uk.gov.hmrc.cataloguefrontend.connector.model.TeamName
+
+@this()
+
+@(
+        repository: GitRepository
```

Comment:
We have two partials that do the same, to support different input types.
Can we change this to take `Seq[Team]` rather than `GitRepository` or `ServiceWithDependency`, then we only need one.
The `isShared` function can be inlined here. Inlining the `Constant.sharedRepoTeamsCutOff` will make it clear that the cutoff is `8`, but the `teamsNames.take` is `15` - which don't align. (`splitAt(8)` and checking if the second list is empty might avoid defining the value more than once)

Reply:
The `GitRepository` object still needs it's `isShared` function and it's used in quite alot of code.  I can raise another ticket to handle this OR we can do it under this ticket?

Reply:
OK - we can leave `isShared`